tests/functional/test_sts.py

```diff
@@ -44,12 +44,12 @@ def test_presigned_url_contains_no_content_type(self):
 
```

Comment:
Looking over `tests/functional/test_credentials` from this PR: https://github.com/boto/botocore/pull/1857/files, it looks like we need to update the test to remove the `sts_regional_endpoints` config variable,